AFQ/data.py

```diff
@@ -392,11 +396,21 @@ def fetch_hcp(subjects,
          "Acknowledgements": """Data were provided by the Human Connectome Project, WU-Minn Consortium (Principal Investigators: David Van Essen and Kamil Ugurbil; 1U54MH091657) funded by the 16 NIH Institutes and Centers that support the NIH Blueprint for Neuroscience Research; and by the McDonnell Center for Systems Neuroscience at Washington University.""",  # noqa
          "Subjects": subjects}
 
-    desc_file = op.join(my_path, 'HCP', 'dataset_description.json')
+    desc_file = op.join(my_path, study, 'dataset_description.json')
     with open(desc_file, 'w') as outfile:
         json.dump(dataset_description, outfile)
 
-    return data_files
+    # Create the BIDS derivatives description file text
```

Comment:
You can also use the new `to_bids_description` function here: https://github.com/yeatmanlab/pyAFQ/blob/4cff21d1fb2ba1b7065d2d5f63e0212c3228fe78/AFQ/data.py#L456

Reply:
Saves a couple of lines of boilerplate.